common/tls/src/main/java/io/helidon/common/tls/spi/TlsManagerProvider.java

```diff
@@ -24,4 +27,17 @@
  */
 public interface TlsManagerProvider extends ConfiguredProvider<TlsManager> {
 
+    /**
+     * Provides the ability to have a unique {@link TlsManager} per unique {@link Config} instance provided.
+     *
+     * @param configBean the config bean instance
+     * @param creator    the creator to apply if not already in cache, which takes the config bean instance
+     * @param <T>        the type of the config bean
+     * @return the tls manager instance from cache, defaulting to creation from the {@code creator} if not in cache
+     */
+    static <T> TlsManager getOrCreate(T configBean,
+                                      Function<Object, TlsManager> creator) {
```

Comment:
Function parameter is not documented here, either document it is the config bean or use supplier

Reply:
pushed